src/libsystemd-network/sd-dhcp6-lease.c

```diff
@@ -587,6 +600,67 @@ int sd_dhcp6_lease_get_captive_portal(sd_dhcp6_lease *lease, const char **ret) {
         return 0;
 }
 
+int dhcp6_lease_add_vendor_option(sd_dhcp6_lease *lease, const uint8_t *optval, size_t optlen, size_t base_offset) {
```

Comment:
`base_offset` is not used??

Reply:
Yes, i was using it earlier. Now I removed it.